CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Version 2.2
 * Changed: Reading Time is displayed only if it is greater than `READING_TIME_LOWER_LIMIT` variable (default 4 min)
 * Changed: Project documentation has been moved inside the Elegant repository
 * Removed: Google Plus author
+* Update FontAwesome to version 4.7.0
```

Comment:
@iranzo this should come under "To Be Released" heading. And please change it to

```
New: FontAwesome updated to version 4.7.0
```